tbtaf/common/result.py

```diff
@@ -57,4 +57,19 @@ def setEndTimestamp(self,endTime):
         self.endTimestamp = endTime
     
     def getEndTimestamp(self):
-        return self.endTimestamp
\ No newline at end of file
+        return self.endTimestamp
+    
+    def to_dict(self):
+        # Convertir los timestamps de datetime a cadena (si están presentes)
+        startTimestamp_str = self.startTimestamp.strftime("%Y-%m-%d %H:%M:%S") if self.startTimestamp else None
+        endTimestamp_str = self.endTimestamp.strftime("%Y-%m-%d %H:%M:%S") if self.endTimestamp else None
+        
+        return {
+            "testVerdict": self.testVerdict,
```

Comment:
As I see these keys being referenced across the code (E.g. resultSource). I do recommend to declate them as contants from the same result class and then consume them in the rest of the classes.